styles.css

```diff
@@ -678,4 +678,548 @@ main hr {
 @media screen and (min-width: 768px) {
   /* Here starts the navbar */
 
-}
\ No newline at end of file
+  .navbar {
+    height: 72px;
+    position: fixed;
+    width: 100%;
+    z-index: 5;
+  }
+
+  .logo {
+    max-width: 200px;
+    height: 40px;
+    margin-left: 0;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 20px;
+    line-height: 20px;
+    cursor: pointer;
+    color: #fff;
+  }
+
+  .logo-link {
+    flex-basis: 50%;
+    margin-left: 10%;
+  }
+
+  #desktop-menu {
+    display: flex;
+    flex-basis: 50%;
+    justify-content: right;
+    color: #fff;
+    height: 100%;
+  }
+
+  .technologies ul li {
+    font-weight: 400;
+    font-size: 22px;
+    line-height: 43px;
+    padding-bottom: 12px;
+    padding-top: 12px;
+    width: 30%;
+  }
+
+  #desktop-ul-menu {
+    display: flex;
+    height: 100%;
+    align-items: center;
+    list-style: none;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  #desktop-ul-menu li {
+    margin-left: 30px;
+    cursor: pointer;
+  }
+
+  .envelope-container {
+    display: flex;
+    height: 100%;
+    align-items: center;
+    margin-left: 30px;
```

Comment:
- [ ] As requested by the previous reviewer, please try to create a class with a `display:flex` rule. You can call it for **example flex-container or d-flex or c-flex or whatever you want** and then add it to any HTML element which should be displayed as a flex. This helps not to repeat the same codes over and over again thereby making your code clean and easy to read. Achieving this makes you an outstanding developer 😉 

This is among the CSS best practices 👇 
![image](https://user-images.githubusercontent.com/90222110/186958299-0a1617ce-582d-4003-a8d3-3e09f4559f79.png)